contracts/libraries/SafeToL2Setup.sol

```diff
@@ -13,10 +13,10 @@ import {SafeStorage} from "../libraries/SafeStorage.sol";
  */
 contract SafeToL2Setup is SafeStorage {
     /**
-     * @notice Address of the contract.
-     * @dev This is used to ensure that the contract is only ever `DELEGATECALL`-ed.
+     * @dev Address of the contract.
+     *      This is used to ensure that the contract is only ever `DELEGATECALL`-ed.
      */
-    address public immutable _SELF;
+    address private immutable _SELF;
 
     /**
      * @notice Event indicating a change of master copy address.
```

Comment:
Same style consistency for `_chainId`?

Reply:
Also, I think the setup function can be made `external`.

Reply:
Removed `_` completely.

Reply:
Ome more comment here you may have missed.

Reply:
Updated 👍🏾